### PR TITLE
adding link to YASP in BigQuery discussion

### PR DESCRIPTION
**Pull request by fhoffa**
@howardchung told me to :)

https://github.com/yasp-dota/yasp/issues/924#issuecomment-197762448

### CODE REVIEW

File: _posts/parsedumps.md [hunk 1]
```diff
@@ -12,3 +12,5 @@ Our motivation for releasing this data is threefold. First, we are committed to
 If you do any interesting ML on this dataset, [let us know](https://github.com/yasp-dota/yasp/issues)! (If you’re looking for more information on the technical specifications of these files, the JSON schema overall format of the parse dumps are explained [here](https://github.com/yasp-dota/yasp/wiki/JSON-Data-Dump). And if you publish or post your findings anywhere, we’d really appreciate it if you cite YASP as your source for the data.) If you seed either of these files for any length of time, thank you! And as always, if you’d like to contribute to the YASP codebase, never hesitate to check out our [github](https://github.com/yasp-dota/yasp).
 
 Happy holidays, everyone :)
+
+*Update 2016-03-17*: [@felipehoffa](https://twitter.com/felipehoffa) loaded the 3.5 million matches dump into Google BigQuery. Learn more about [querying YASP on BigQuery](https://github.com/yasp-dota/yasp/issues/924) on our github.
```
Comment (howardchung) [inline _posts/parsedumps.md]: GitHub.

Also consider just directly linking the table.

Comment (fhoffa) [inline _posts/parsedumps.md]: Done (also capitalized previous github mention)

I'd skip linking to the table for now, as this is only _v1... hopefully we'll come up with a better table structure depending on needs, and update the GitHub thread accordingly.

**Comment by howardchung** [comment]
Do you want to add the https://bigquery.cloud.google.com/table/fh-bigquery:public_dump.dota2_yasp_v1 link?

**Comment by howardchung** [comment]
signoff, @albertcui please review and merge

**Comment by albertcui** [comment]
Looks cool :) Thanks!

**Comment by jiehu567** [comment]
This BigQuerry is awesome!
Can I use the data for some research analysis? 
I'm Jie, a statistician for gaming. Can I have some description of attributes like: tower_status_xx, parse_status? I can guess out meaning of most attributes.

**Comment by albertcui** [comment]
Some documentation here:
https://github.com/yasp-dota/yasp/wiki/JSON-Data-Dump

**Comment by jiehu567** [comment]
thank you so much!

**Pull request merged**